pkg/plugins/platform/utils.go

```diff
@@ -197,3 +210,15 @@ func ddevContainersRunning() (bool, error) {
 	}
 	return false, nil
 }
+
+// exists returns whether the given file or directory exists or not
+func exists(path string) (bool, error) {
```

Comment:
Please use the FileExists from our fileutils package: https://github.com/drud/ddev/blob/master/pkg/fileutil/files.go#L119

It's already imported here, so it should be an easy update?